test/fs.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE
+'use strict';
+
+const {getConfigFromFile} = require('../lib/utils/fs');
+const packageJSON = require('../package.json');
+const expectations = require('./fixtures/run-expectations');
+
+describe('Config', () => {
+  describe('validate input', () => {
+    context('from package.json', () => {
+      it('should return empty object if pwmetrics property not defined', () => {
+        expect(getConfigFromFile()).to.be.deep.equal({});
+      });
+    });
+    describe('with pwmetrics property defined', () => {
```

Comment:
Context is an alias for the describe function, I used it to increase readability and not have nested describes. I could change for consistency, might be best...